p02_7a.py

```diff
@@ -0,0 +1,16 @@
+import random
+
+def coinflip():
+	s = random.random()
+	if float(s) < 0.5:
```

Comment:
This is good. But you can also try one-line version. See `condition ? true_action : false_action`